web_hosting_manager/app/safenet_comm/temp.js

```diff
@@ -1,11 +1,10 @@
 import temp from 'temp';
 
-const tempTrack = temp.track();
```

Comment:
Are we sure we don't want to invoke `temp.track()` and explcitly removing it? 
From looking at the doc it seems that's to automatically track and cleanup files at exit: https://www.npmjs.com/package/temp